src/pages/content/punctuation.mdx

```diff
@@ -1,12 +1,99 @@
 ---
 title: Punctuation
-description: >
-  Lorem ipsum dolor sit amet, consectetur adipiscing
-  elit, sed do eiusmod tempor incididunt ut labore et
-  dolore magna aliqua.
 ---
 
-Coming soon
+## Ampersand
+
+Avoid ampersands in place of “and.” Don’t use “+” unless you’re opening a coffee shop in Portland.
+
+## Apostrophe
+
+Use apostrophes to denote possession. Avoid apostrophes to denote plurals even for initialisms.
+
+## Comma
+
+Always use the Oxford comma, also known as the serial comma, to precede the final item in a list.
+Don’t use commas in buttons.
+
+## Colon
+
+In general, avoid using colons in product, especially when describing some attribute and its value.
+Use emphasis styling instead to distinguish between these two types of content.
+
+It is OK to use a colon to introduce a bulleted list. When doing so, make sure you don’t split a
+sentence at the colon. This causes challenges for translation.
+
+<BestPractice>
+  <Do>
+    <p>
+      <b>Name</b> Omar Bongo
+    </p>
+  </Do>
+  <Dont>
+    <p>Name: Omar Bongo</p>
+  </Dont>
+</BestPractice>
+
+## Ellipse
+
+Garden creates ellipses automatically when words are truncated. Don’t add them manually to the UI.
+
+## Exclamation point
+
+These are too loud for most of our brand tones. We tend not to use them at all.
+
+## Hyphens and dashes
+
+Err on the side of fewer hyphens and dashes. Use en dashes with no spaces to communicate a range.
+Avoid using en or em dashes to communicate a break in your thoughts.
+
+_This plan affects any customers with 100–150 active agents._
+
+## Parentheses
+
+Use sparingly. Use parentheses to introduce an acronym in product and never put crucial information
+in parentheses.
+
+## Period
+
+Use periods to end full sentences in body copy. Consult this handy chart to find out where we use
+periods.
+
+| Component                | Does it need a period?                                                                                                                                                                          |
+| ------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Breadcrumbs              | No. Bullets should be sentence fragments. If a bullet has more than one sentence, use closing punctuation on each. If a bullet has more than two sentences, it should be a description instead. |
+| Bullets (in a list)      | No                                                                                                                                                                                              |
```

Comment:
Looks like the Bullets answer is currently showing no, and the Breadcrumbs answer is currently showing the Bullet answer.